tests/demos_test.go

```diff
@@ -0,0 +1,39 @@
+package tests_test
+
+import (
+	"io/ioutil"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	yaml "github.com/ghodss/yaml"
+
+	k8s "k8s.io/api/apps/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+var _ = Describe("Demo", func() {
```

Comment:
maybe let's keep every demo test in its own module? ginkgo/gomega tests grow rather fast and it is hard (at least for me) to find what I'm looking for in them